apps/api/src/api-key/service/api-key.service.ts

```diff
@@ -146,7 +147,7 @@ export class ApiKeyService {
         }
       },
       skip: page * limit,
-      take: limit,
+      take: limitMaxItemsPerPage(limit),
```

Comment:
Could you please add the pagination response in here? I think the previous contributor missed it.